driver/pipeline/acquisition.cpp

```diff
@@ -153,7 +153,9 @@ void Acquisition::runInternal() {
     // A running breaker means the pipeline user has not called stop.
     while (this->breaker.running()) {
         auto [frame, source_err] = this->source->read(this->breaker);
-        // LOG(INFO) << "Read frame from source";
+        if(!this->middleware_chain.empty()){
```

Comment:
Fix formatting. Also why can't you just call exec on the empty chain? I feel like this if statement should be internal to the middleware.